client/my-sites/upgrades/checkout-thank-you/business-plan-details.jsx

```diff
@@ -7,26 +7,31 @@ import React from 'react';
  * Internal dependencies
  */
 import i18n from 'lib/mixins/i18n';
+import { isBusiness } from 'lib/products-values';
 import PurchaseDetail from 'components/purchase-detail';
 
-const BusinessPlanDetails = ( { selectedSite } ) => {
+const BusinessPlanDetails = ( { selectedSite, sitePlans } ) => {
+	const plan = find( sitePlans.data, isBusiness );
```

Comment:
Thanks for catching this - I think the linter/bundler miss this because of the presence of `window.find`, so I've missed this a couple times.